components/spin/style/index.tsx

```diff
@@ -57,6 +58,18 @@ const genSpinStyle: GenerateStyle<SpinToken> = (token: SpinToken): CSSObject =>
       opacity: 1,
     },
 
+    '&-fullscreen': {
+      position: 'fixed',
+      width: '100vw',
+      height: '100vh',
+      background: 'rgba(0,0,0,0.2)',
```

Comment:
```suggestion
      backgroundColor: token.colorBgMask,
```

Reply:
I thought in use that, but it feel a little too much dark, do we have a lighter one? or, can I create a component token?

![image](https://github.com/ant-design/ant-design/assets/27746333/8239c614-8c6a-46d8-9b49-1d783be8aa61)

Reply:
Yes, you can create custom token, but this is not the final style, we still need to wait for the discussion results from the designers.